pkg/northbound/gnmi/set.go

```diff
@@ -184,6 +183,7 @@ func (s *Server) buildUpdateResults(targetUpdates mapTargetUpdates,
 		changeID, configName, cont, err := setChange(target, version, updates, targetRemoves[target])
 		//if the error is not nil and we need to continue do so
 		if err != nil && !cont {
+			//TODO Rollback needs also to happen here.
```

Comment:
you took this back out?

Reply:
yes, needs further investigation, want to get these fixes in ASAP